src/status_im/contexts/wallet/common/sheets/account_options/view.cljs

```diff
@@ -32,12 +32,13 @@
 
 (defn- options
   [{:keys [theme show-account-selector? options-height]}]
-  (let [{:keys [name color emoji address watch-only?]} (rf/sub [:wallet/current-viewing-account])
-        network-preference-details                     (rf/sub [:wallet/network-preference-details])
-        multichain-address                             (utils/get-multichain-address
-                                                        network-preference-details
-                                                        address)
-        share-title                                    (str name " " (i18n/label :t/address))]
+  (let [{:keys            [name color emoji address watch-only?]
+         default-account? :wallet} (rf/sub [:wallet/current-viewing-account])
+        network-preference-details (rf/sub [:wallet/network-preference-details])
+        multichain-address         (utils/get-multichain-address
+                                    network-preference-details
+                                    address)
+        share-title                (str name " " (i18n/label :t/address))]
```

Comment:
this is better as an i18n label with a param.
e.g  `(i18n/label :t/address {:name name})`

Reply:
It's not related to my issue, but I will fix it)))